connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -113,6 +114,9 @@
 
     private final ExecutorService forwardRequestExecutor;
 
+    private Map<String, ConfigDef> configs = new HashMap<>();
+    private Map<String, Connector> tempConnectors = new HashMap<>();
```

Comment:
I think this is just leftover? I don't see any reason we'd need thos to replace the one in `AbstractHerder`